homeassistant/components/conversation/__init__.py

```diff
@@ -48,7 +50,7 @@
 
 @core.callback
 @bind_hass
-def async_set_agent(hass: core.HomeAssistant, agent: AbstractConversationAgent):
+def async_set_agent(hass: core.HomeAssistant, agent: AbstractConversationAgent | None):
```

Comment:
I have assumed that the error is inside `conversation` integration, but maybe the error is actually in `almond` integration which shouldn't be setting it to `None`?

Reply:
Yeah, that is fine, it will initialize the default agent kicking the `intent` integration into existence.